web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -132,9 +152,7 @@ export class StylesheetManager {
         if(this.doCacheBusting) {
           ttf = this.cacheBust(ttf);
         }
-        s=s+'src:url(\''+ttf+'\') format(\'truetype\');';
-      } else {
-        return null;
+        source = "url('"+ttf+"') format('truetype')";
       }
     } else {
       var s0 = [];
```

Comment:
```suggestion
```

Is `s0` used any more?

Reply:
... I thought I did remove it; good catch.